python/tests/res/analysis/test_std_enkf_debug.py

```diff
@@ -14,22 +14,31 @@
 #  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
 #  for more details.
 import sys
+import os
 
 import res
+
```

Comment:
white space?

Reply:
intentionally, from pep:
>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.

Reply:
was intentional, but the library isn't actually used, so I'll remove the import